stan/math/prim/prob/lkj_cov_lpdf.hpp

```diff
@@ -12,87 +13,83 @@ namespace math {
 // LKJ_cov(y|mu, sigma, eta) [ y covariance matrix (not correlation matrix)
 //                         mu vector, sigma > 0 vector, eta > 0 ]
 template <bool propto, typename T_y, typename T_loc, typename T_scale,
-          typename T_shape>
-return_type_t<T_y, T_loc, T_scale, T_shape> lkj_cov_lpdf(
-    const Eigen::Matrix<T_y, Eigen::Dynamic, Eigen::Dynamic>& y,
-    const Eigen::Matrix<T_loc, Eigen::Dynamic, 1>& mu,
-    const Eigen::Matrix<T_scale, Eigen::Dynamic, 1>& sigma,
-    const T_shape& eta) {
+          typename T_shape, require_eigen_matrix_t<T_y>* = nullptr,
+          require_all_eigen_col_vector_t<T_loc, T_scale>* = nullptr>
+return_type_t<T_y, T_loc, T_scale, T_shape> lkj_cov_lpdf(const T_y& y,
```

Comment:
Also can this function be merged with the one below it using the scalar broadcasting stuff?

Otherwise location and scale both need to be column vectors or both scalars.

Reply:
They were written before version 1.0 and disappeared.

Reply:
@t4c1 Just mark these functions deprecated and they're fine as they are as long as they pass tests then.

Reply:
done